examples/low_light_enhancement/supervised/nafnet/train_nafnet.ipynb

```diff
@@ -0,0 +1,143 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import wandb\n",
+    "from wandb.keras import WandbMetricsLogger\n",
+    "\n",
+    "from tqdm.keras import TqdmCallback\n",
+    "\n",
+    "import tensorflow as tf\n",
+    "tf.get_logger().setLevel('ERROR')\n",
+    "\n",
+    "from restorers.model import NAFNet\n",
+    "from restorers.dataloader import LOLDataLoader\n",
+    "from restorers.losses import CharbonnierLoss, PSNRLoss\n",
+    "from restorers.metrics import PSNRMetric, SSIMMetric\n",
+    "from restorers.utils import get_model_checkpoint_callback"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "wandb.init(project=\"nafnet\", entity=\"ml-colabs\")\n",
+    "\n",
+    "data_loader = LOLDataLoader(\n",
+    "    image_size=128,\n",
+    "    bit_depth=8,\n",
+    "    val_split=0.2,\n",
+    "    visualize_on_wandb=False,\n",
+    "    dataset_artifact_address=\"ml-colabs/dataset/LoL:v0\"\n",
+    ")\n",
+    "\n",
+    "train_dataset, val_dataset = data_loader.get_datasets(batch_size=4)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "model = NAFNet()\n",
```

Comment:
Just a question, we don't want to use model_configs that you defined earlier here too? They are used in the .py file.

Reply:
Which model configs? We're currently using the default ones.